config.md

```diff
@@ -139,6 +139,40 @@ For POSIX platforms the `mounts` structure has the following fields:
 ]
 ```
 
+### <a name="configLinuxMounts" />Linux Mounts
+
+For Linux, the `mounts` structure mounts MAY have the following additional fields:
```

Comment:
We should probably describe what the expected behaviour is of the runtime if a spec includes attr's that are not supported by the kernel; should it ignore those attributes, or should it fail?

Reply:
Added `The runtime MUST [generate an error](runtime.md#errors) for any values which cannot be mapped to a relevant kernel interface.`